packages/mobile/e2e/src/usecases/ExchangeCelo.js

```diff
@@ -53,10 +52,15 @@ export default ExchangeCelo = () => {
     await enterPinUiIfNecessary()
     // Get transaction time
     let transactionTime = new Date()
-    // Scroll to top of feed
-    await element(by.id('ExchangeScrollView')).scroll(200, 'down')
     // Wait 10 seconds checking that error banner is not visible each second
     await waitForExpectNotVisible('errorBanner')
+    // Scroll to top of feed - scroll not possible if not all transactions loaded
+    try {
+      await waitFor(element(by.id('ExchangeScrollView')))
+        .toBeVisible()
+        .withTimeout(10 * 1000)
+      await element(by.id('ExchangeScrollView')).scroll(200, 'down')
+    } catch {}
```

Comment:
do we need the try/catch here? it seems that if anything fails inside the try, we will do nothing in the catch and then proceed to the next step (which should fail if the previous steps didn't complete, i assume?). maybe i am missing something but that seems like the same flow as if the try/catch was not there?

Reply:
We want to try and wait for the element and scroll; however, it fails we don't want the test to fail. This is needed because if the entire transaction feed loads we want to scroll a bit, if the transaction feed doesn't load the transaction will be at the top and we can assert on that.